### PR TITLE
Created Lookml Dashboard

### PR DESCRIPTION
**Pull request by Krsrinidhi**

### CODE REVIEW

**Review by tarasovv1 — COMMENTED** [review]
Please, add a "Header with view Purpose" to job_agg_rfn.view.lkml, follow the style proposed here: https://g3doc.corp.google.com/cloud/solutions/cortex/g3doc/lookml_best_practices.md#general-guidelines-views

Please, mark JobID field as a PK  in the job_agg_rfn.view.lkml: https://g3doc.corp.google.com/cloud/solutions/cortex/g3doc/lookml_best_practices.md#primary-key-guidelines

**Comment by Krsrinidhi** [comment]
@tarasovv1 Primary key is already defined in jobagg_rfn

2. Added purpose and details

**Review by pthoutam-googler — APPROVED** [review]

**Review by tarasovv1 — APPROVED** [review]

**Review by thjennifer — APPROVED** [review]
I don't think the multiple comments describing what a dimension and measure are in general are necessary and should be removed during the next set of changes. 

Do though include any comments for a specific dimension or measure that helps explain it's meaning or use. For example:

```
# Email Broadcast concatenates email and job id so that when this dimension is used as a filter in a dashboard or explore, the user can search for either value.
```

File: dashboards/SFMC.dashboard.lookml [hunk 1]
```diff
@@ -0,0 +1,2003 @@
+- dashboard: salesforce_marketing_cloud_email_broadcast_performance
```
Comment (thjennifer) [inline dashboards/SFMC.dashboard.lookml:1]: @Krsrinidhi 

In next round of edits, please consider the following for this LookML Dashboard:

Also, in the LookML Dashboard:

- Remove each `model: cortex-sfmc` line used (will be one for each tile). When no model is specified, Looker will use the model the dashboard is included in.
- Be sure to Name each tile a unique title and name (avoid generic  - title: New Tile  -name: New Tile)
- When using the Advanced Viz Config, you only have to specify the parts that are altered/changed. There's over 500 lines just on the Advanced Viz Config alone so make sure to keep only the part you edited (i.e., if you edited the tooltip, just include the tooltip part). It's a little tricky though to capture the right json structure. I'll look for some documentation and examples to share.

**Pull request merged**